### PR TITLE
New Port Skeleton Boomerang

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Skeleton Boomerang

## Game Information
- **Title**: Skeleton Boomerang
- **URL**: https://store.steampowered.com/app/658150/Skeleton_Boomerang/

## Submission Requirements
CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] armHF
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/skeletonboomerang/cover.png
Comment (cdeletre) [inline ports/skeletonboomerang/cover.png:1]: skeletonboomerang-fs8

;)

File: ports/skeletonboomerang/skeletonboomerang/libs/libcrypto.so.1.1
Comment (cdeletre) [inline ports/skeletonboomerang/skeletonboomerang/libs/libcrypto.so.1.1:1]: I know we don't like administrative tasks ;)
https://github.com/cdeletre/PortMaster-MiniDoom2/blob/main/minidoom2/LICENSE.libcrypto.txt

File: ports/skeletonboomerang/skeletonboomerang/libs/libzip.so.4
Comment (cdeletre) [inline ports/skeletonboomerang/skeletonboomerang/libs/libzip.so.4:1]: https://github.com/cdeletre/PortMaster-MiniDoom2/blob/main/minidoom2/LICENSE.libzip.txt

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/skeletonboomerang/port.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "skeletonboomerang.zip",
+  "items": [
+    "Skeleton Boomerang",
+    "skeletonboomerang"
```
[the remark this answers is not in the document]
Comment (Fraxinus88) [reply ports/skeletonboomerang/port.json:6]: Whoopsies

File: ports/skeletonboomerang/cover.png
Comment (Fraxinus88) [reply ports/skeletonboomerang/cover.png:1]: Awesome!

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**